app/_locales/en/messages.json

```diff
@@ -1140,6 +1140,12 @@
   "searchTokens": {
     "message": "Search Tokens"
   },
+  "searchAccounts": {
```

Comment:
I don't know how you approach translations so let me know if I need manually translate to all locales

Reply:
Nope, this is OK 👍. While it would be great to get translations, that's not something we'd expect contributors to be able to do.